ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml

```diff
@@ -41,10 +41,11 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
     workflow: cucushift-installer-rehearse-aws-ipi-private
 - as: installer-rehearse-gcp
+  cluster: build05
```

Comment:
Do we have to limit the testing on the specific cluster?

Reply:
No. By default it is build02. But it will report [this error](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/33009/rehearse-33009-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.11-installer-rehearse-gcp/1580204342491746304) . Per [this slack thread](https://coreos.slack.com/archives/CBN38N3MW/p1665587230605979?thread_ts=1665586834.890709&cid=CBN38N3MW), they are moving from build02 to others. I tried build01, not worked; so updated to build05 instead.